src/View/mainpage/DrawROIWindow/Drawing.py

```diff
@@ -115,15 +117,44 @@ def _display_pixel_color(self):
                 colors = QColor(c).getRgbF()
                 self.according_color_dict[(x_coord, y_coord)] = colors
 
-            color = QtGui.QColor()
-            color.setRgb(90, 250, 175, 200)
-            points = get_pixel_coords(self.according_color_dict, self.rows,
-                                      self.cols)
-            for x_coord, y_coord in points:
-                self.q_image.setPixelColor(x_coord, y_coord, color)
+            points = get_pixel_coords(self.according_color_dict, self.rows, self.cols)
+
+            self._set_color_of_pixels(points)
 
             self.refresh_image()
 
+    def _set_color_of_pixels(self, points):
+        """
+        Takes a set of points that need to be coloured, and updates the colour of
+        each pixel with the drawn on colour, blended together based on the transparency value.
+
+        :param points:  Set of x and y co-ordinates of pixels to be coloured
+        """
+        logging.debug("_set_color_of_pixels started")
```

Comment:
Given what was discussed in the meeting should I keep these logging statements in? Possibly comment out?

Reply:
Refer to Ryan / Ashley input

Reply:
Debug statements will stay in